here/here_aggregation.md

```diff
@@ -96,13 +96,13 @@ Common output parameters:
 
 - Minimum sample size: Depending on the extent of the study area and the time range requests, we have to ensure we are aggregating enough data to estimate travel times, usually a minimum of a month of data.
 - Harmonic mean: Harmonic mean has to be used when averaging speed, or we can average travel time or travel time index with the arithmetic mean.
-- Links without data: To estimate segment level travel time when some links don't have data, we only include segments where at least 80% of links (by distance) has observations.
+- Links without data: To estimate segment level travel time when some links don't have data, we only include segments where at least 80% of links (by distance) have observations. 
 
 ## Aggregation
 
 ### Using congestion tables
 
-Congestions summary tables are updated by an [airflow pipeline](https://github.com/CityofToronto/bdit_data-sources/blob/secret_dags/dags/congestion_refresh.py) that runs every day and aggregates 5-min bin link level data up to segment level, creating segment weekly travel time index.
+Congestion summary tables are updated by an [airflow pipeline](https://github.com/CityofToronto/bdit_data-sources/blob/secret_dags/dags/congestion_refresh.py) that runs every day and aggregates 5-min bin link level data up to segment level, creating segment weekly travel time index.
 
 When to use congestion summary tables:
```

Comment:
ahhh I should update the congestion summary table part, this is outdate :(

Reply:
updated..... for meow, will need to remove the second part once we officially move everything to the new congestion network aggregation